### DISCUSSION TITLE
/posts/[slug].js could not be found in Production vercel deploy/vercel --prod

### DISCUSSION THREAD

**Discussion by kmocorro** [vercel/next.js#18607] [Help]
# Bug report
/posts/[slug].js returns 404 | This page could not be found in Production vercel deploy/vercel --prod/github integration.

## Describe the bug
A statically generated blog example using Next.js and Prismic.  https://github.com/vercel/next.js/tree/canary/examples/cms-prismic
is not working properly in production deployment. In local, somehow works.

## To Reproduce
1.) Follow the instruction stated in the tutorial: https://github.com/vercel/next.js/tree/canary/examples/cms-prismic
2.) Add env variables using prismic token/ repo name that you'll be created in prismic.io
3.) Then, deploy in vercel.
4.) See Error on clicking one of your "posts" from /posts/[slug].js
![image](https://user-images.githubusercontent.com/26828995/97797232-65771400-1c56-11eb-8aed-28fdfc891fc7.png)

## Expected behavior
It should return "loading..." then and generate static page based on the slug data. (this works while npm run dev)

## Screenshots
![image](https://user-images.githubusercontent.com/26828995/97797288-ca326e80-1c56-11eb-9a9f-e3e8883b64ba.png)
![image](https://user-images.githubusercontent.com/26828995/97797293-d3234000-1c56-11eb-92c2-0f3f96ce8025.png)

## System information

- OS: [e.g. Windows]
- Browser (if applies) [e.g. firefox dev, firefox, chrome, safari]
- Version of Next.js: [e.g. 10.0.0]
- Version of Node.js: [e.g. 12.*]

**Comment by timneutkens** [comment]
Looks like the correct page is being served, just that your data fetching does not return a `ipsum-lorem` post 🤔 Will convert this into a help discussion given it's not a bug.
